src/api/status/env.local

```diff
@@ -1,4 +1,5 @@
 PLANET_PORT=1111
 PATH_PREFIX=/v1/status
 API_HOST=localhost
-API_URL= http://localhost:3000
+POSTS_URL=http://localhost/v1/posts
+MOCK_REDIS=1
```

Comment:
This is probably fine.  The only risk of doing this is that we won't be able to connect to a live Redis instance when running this way, but I think that's acceptable, since you can do it via docker.